gitops/ckcp/base/deployment.yaml

```diff
@@ -16,8 +16,19 @@ spec:
     spec:
       serviceAccountName: anyuid
       containers:
-        - image: ckcp
+        - image: ghcr.io/kcp-dev/kcp:976a437
           name: ckcp
+          command: [
+            "/bin/sh",
+            "-c",
+            "rm -rf .kcp;
+            ./kcp/bin/kcp start \
+              --push-mode=true \
+              --pull-mode=false \
+              --run-controllers \
+              --auto-publish-apis \
+              --resources-to-sync=deployments.apps,statefulsets.apps,pods,services,secrets,persistentvolumeclaims"
```

Comment:
With phase 1 approach we want to sync more, starting with:
ingresses.networking.k8s.io,deployments.apps,services,conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,runs.tekton.dev,tasks.tekton.dev,taskruns.tekton.dev

we may letter add resource types for other controllers

Reply:
Ideally we would notset those resources via the command line, but register them later through KCP, so that we use the same behavior in local dev and on staging. So I'm in favor of reducing this list instead of increasing it.

Reply:
We need first to have something that works with the current state of the world.
When things have progressed in kcp and the syncer pull mode is available we will need to rework it by any way